Poker.BE/Poker.BE.API/Controllers/AuthenticationController.cs

```diff
@@ -7,7 +10,69 @@
 
 namespace Poker.BE.API.Controllers
 {
-    public class AuthenticationController : ApiController
-    {
-    }
+	public class AuthenticationController : ApiController
+	{
+		#region Fields
+		private IAuthenticationService service;
+		#endregion
+
+		#region Constructors
+		public AuthenticationController()
+		{
+			service = new Service.Services.AuthenticationService();
+		}
+		#endregion
+
+		#region Methods
+		[HttpPost]
+		public HttpResponseMessage Login(LoginRequest request)
+		{
+			var result = default(LoginResult);
+
+			try
+			{
+				result = service.Login(request);
+			}
+			catch (Exception e)
+			{
+				result.ErrorMessage = e.Message;
```

Comment:
by this way you overwrite the `ErrorMessage` of the result by this new error message. i need to check what's up now with the property special setter i made for this `ErrorMessage` property. i wanted to make it `AccumelatorSetter` so we can just do regular assignment like this `=` instead of `+=`.